Framework: source-map dependency bug

#### Steps to reproduce
1. Clone `wp-calypso` repository: `> git clone git@github.com:Automattic/wp-calypso.git`
2. Go to wp-calypso folder: `> cd wp-calypso`
2. Run the app: `> make run`

#### What I expected

Calypso woking locally.


#### What happened instead
I get source-map dependency bug:

```
ERROR in Cannot find module 'source-map'
 @ ./client/components/tinymce/index.jsx 78:14-40
```

#### OS version
OSX - El Capitan `10-11-3`

node version: `4.3.0`
npm version: `2.14.12`

`nodejs` and `npm` releases: https://nodejs.org/en/download/releases/

#### Screenshot

![image](https://cloud.githubusercontent.com/assets/77539/13675092/d1c5a6c6-e6be-11e5-9993-f611495d7b48.png)

--- a/package.json
+++ b/package.json
@@ -92,6 +92,7 @@
     "redux-thunk": "1.0.0",
     "rtlcss": "1.6.1",
     "sanitize-html": "1.11.1",
+    "source-map": "0.1.39",
     "source-map-support": "0.3.2",
     "srcset": "1.0.0",
     "store": "1.3.16",

